src/constructs/cloudwatch/no-monitoring.ts

```diff
@@ -0,0 +1,3 @@
+export interface NoMonitoring {
```

Comment:
Not too certain on the naming or placement of this interface. It looks like it is only ever used in conjunction with `GuLambdaErrorPercentageMonitoringProps`? Maybe we can colocate them?

Reply:
These were previously colocated, but this PR has refactored based on this suggestion: https://github.com/guardian/cdk/pull/186#discussion_r563738361. It's slightly preemptive, but we are expecting to share this object when we add different types of monitoring.

Reply:
Ah ok, happy to leave then.